reloj-checador/Gemfile

```diff
@@ -49,10 +49,15 @@ group :test do
   gem 'capybara', '>= 3.26'
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
+  gem 'database_cleaner-active_record'
```

Comment:
Only as a comment, this gem isn't necessary now that rails cleans the data created during each test by default via transactions test, https://guides.rubyonrails.org/testing.html#testing-parallel-transactions

Reply:
I'll consider this